plugins/outputs/whatap/whatap.go

```diff
@@ -0,0 +1,183 @@
+package whatap
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/outputs"
+
+	wio "github.com/whatap/go-api/common/io"
+	wpack "github.com/whatap/go-api/common/lang/pack"
+	whash "github.com/whatap/go-api/common/util/hash"
+)
+
+const (
+	NetSrcAgentOneway  = 10
+	NetSrcAgentVersion = 0
+	Scheme             = "tcp"
```

Comment:
I don't think you need to export those constants, do you?

Reply:
Edited and applied. We ask for confirmation.